modules/installation-mirror-repository.adoc

```diff
@@ -71,10 +71,7 @@ To use the new mirrored repository to install, add the following section to the
 imageContentSources:
 - mirrors:
   - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
-  source: quay.io/openshift-release-dev/ocp-release
-- mirrors:
-  - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
-  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+  source: <upstream_repository>
```

Comment:
@wking, I'd add an annotation to this line to explain how these upstream repositories might vary. I'd also add a note after the next step to emphasize that they must use their command output.